ONSdigital/dp-frontend-filter-flex-dataset
# Feature/use dp-renderer

### What

- Updated go version to `1.17`
- Instantiated `dp-renderer`

### How to review

- Sense check
- Tests pass
- Pull local branch and `make debug`
  - `npm run dev` on `dp-design-system` 
  - Go to http://localhost:20100/flex/helloworld
  - `overview` template should display with ONS' header and footer

![image](https://user-images.githubusercontent.com/19624419/148963027-6fd2ed98-090d-47bf-ac9a-1e134e60aac8.png)

### Who can review

Anyone but me
